app.css

```diff
@@ -1,3 +1,287 @@
+  min-height: 100vh;
+  font-family: 'PT Sans', sans-serif;
+  /*font-weight: 400;*/
+
+
+}
+/* Top Navigation */
+.top-nav {
+  overflow: hidden;
+  height: 3rem;
+  background: #5185aa;
+  margin: 0 2px;
+}
+.top-nav h2 {
+  position: absolute;
+  display: inline-block;
+  font-size: 14px;
+  margin-left: 20px;
+}
+.surf-logo {
+  height: 30px;
+  width: 30px;
+  margin: 5px 0 0 100px;
+}
+
+.top-nav ul {
+  float: right;
+  margin-right: 2rem;
+}
+.top-nav li {
+  display: inline;
+  margin: 0 0.8em 0 0;
+  list-style: none;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: bold;
+  color: black;
+}
+
+.top-nav a {
+  text-decoration: none;
+  color: black;
+}
+
+
+/* Main */
+main {
+  overflow: auto;
+  flex: 1 0;
+  /*background: #999;*/
+}
+
+header {
+  overflow: hidden;
+  position: relative;
+  height: calc(100vh - 5rem);
+  /*max-width: 100%;*/
+  background: url("http://showsomescents.com/wp-content/uploads/2014/01/Oceans-mist-waves.jpg")
+    no-repeat center center;
+  background-size: cover;
+  margin: 0 2px;
+}
+
+.hero {
+  position: absolute;
+  transform: translateX(50%);
+  font-size: 2.2rem;
+  color: white;
+}
+header h3 {
+  display: block;
+  position: absolute;
+  margin-left: 50%;
+  text-align: center;
+  bottom: 44%;
+  font-size: 12px;
+  color: black;
+
+}
+/* Middle Content */
+.left-column {
+  display: inline-block;
+  width: 50%;
+  margin: 10px 0 10px 100px;
+  padding: 10px 0 2px 10px;
+  background: white;
+}
+
+.author-pic {
+  height: 15px;
+  width: 15px;
+  border-radius: 100%;
+}
+.left-column h1 {
+  font-size: 3rem;
+  font-weight: bold;
+  color: #23252a;
+}
 
+.left-column h2 {
+  border-left: 2px lightgrey solid;
+  padding: 20px 20px 0 20px;
+  margin-bottom: 0;
+  color: #5185aa;
+}
+.left-column span {
+  margin-top: 0;
+  color: #5185aa;
+}
+.left-column ul {
+  float: left;
+  margin-left: 0;
+  line-height: normal;
+  text-decoration: none;
+  list-style: none;
+}
+.left-column li {
+  display: inline;
+  width: 100%;
+  padding: 0 10px;
+  font-size: 11px;
+}
+.left-column .fa-comment, .fa-cloud-upload {
+  border: 2px darkgrey solid;
+  background: darkgrey;
+  padding: 2px;
+  color: white;
+}
+
+.left-column .fa-share-square-o {
+  border: 2px black solid;
+  background: black;
+  padding: 2px;
+  color: white;
+}
+
+.left-column .comments-box a {
+  text-decoration: none;
+}
+
+.sidebar {
+  float: right;
+  width: 33%;
+  margin: 34px 50px 0 8px;
+  padding: 10px;
+  background: white;
+}
+
+.sidebar .author-info {
+  text-align: center;
+  color: #5185aa;
+  padding: 0;
+  margin-top: 0;
+}
+.sidebar .ad a {
+  margin-right: 28px;
+  font-size: 11px;
+  color: lightgrey;
+}
+.sidebar-section h3 {
+  display: block;
+  margin: 64px 0 0 24px;
+  font-size: 22px;
+  font-weight: bold;
+  color: #5185aa;
+}
+.sidebar-section p {
+  display: block;
+  margin: 0 0 0 24px;
+  font-size: 14px;
+  color: grey;
+
+}
+.author-image {
+  height: 40px;
+  width: 40px;
```

Comment:
If the image given is square, then you will only have to set the `width` not the height. This avoids the chance that you may stretch the image.